src/components/app/app.js

```diff
@@ -35,21 +40,31 @@ export class AppComponent {
       onAboutClick: this.handleAboutOnclick.bind(this)
     });
     this.toolBarComponent.mount();
-    this.drawerComponent = new DrawerComponent(this.drawerMountPoint, {
-      onAddAccountClick: this.handleAddAccountClick.bind(this),
-      onAddTagClick: this.handleAddTagOnclick.bind(this)
-    });
-    this.drawerComponent.mount();
-    this.transactionsComponent = new TransactionsComponent(
-      this.transactionsMountPoint
-    );
-    this.transactionsComponent.mount();
-    this.addAccountDialog = new AddAccountComponent(this.addAccountMountPoint);
-    this.addAccountDialog.mount();
+    this.initDrawer();
     this.addTagDialog = new AddTagComponent(this.addTagMountPoint);
     this.addTagDialog.mount();
     this.aboutDialog = new AboutComponent(this.aboutMountPoint);
     this.aboutDialog.mount();
+    this.mountTransactionsComponent();
+  }
+
+  mountTransactionsComponent(accounts) {
+    this.transactionsComponent = new TransactionsComponent(
+      this.transactionsMountPoint,
+      { accounts: accounts }
```

Comment:
you don't need to send accounts to this component, so you don't need this method. please, restore rows 43-46